### ISSUE TITLE
[Suggestion] Replace MCEdit with Amulet Map Editor for 1.12.2+

### ISSUE THREAD

**Issue by business-goose** [MultiMC/MultiMC5#3713]
### Role

Sometimes, I incorrectly type a command in a command block and it can get very difficult to delete them.

### Suggestion

MultiMC should substitute the MCEdit editor with Amulet for post 1.12.2 versions.

### Benefit

If you open a world with MCEdit (at least on Windows) and it is post 1.12.2 it is impossible to get anything done with it.

### This suggestion is unique

I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by AverageComet250** [comment]
Amulet integration would be very useful, especially since MCedit is pretty much deprecated.

**Comment by spannerman79** [comment]
> Amulet integration would be very useful, especially since MCedit is pretty much deprecated.

And yet Amulet is useless for anything lower then 1.12.x

So no, in this case Amulet is just as useful as tits on a bull in this case.

It is arrogant to think that MC versions > 1.12.x is better then any other versions considering that many still play older versions of MC.

**Comment by business-goose** [comment]
> > Amulet integration would be very useful, especially since MCedit is pretty much deprecated.
> 
> And yet Amulet is useless for anything lower then 1.12.x
> 
> So no, in this case Amulet is just as useful as tits on a bull in this case.
> 
> It is arrogant to think that MC versions > 1.12.x is better then any other versions considering that many still play older versions of MC.

All he said was that MCEdit is pretty much deprecated, which it arguably is. He never said that he perferred 1.12.x+. Please don't create arguments out of nothing.

**Comment by spannerman79** [comment]
It wasn't an argument - it was a statement of fact. 

If you have the issue of working out the difference that is something you need to work on.

**Comment by business-goose** [comment]
> It wasn't an argument - it was a statement of fact.
> 
> If you have the issue of working out the difference that is something you need to work on.

Once again, please don't try to create controversy over literally nothing. Please don't assume that me, or anybody else for that matter, only plays on 1.12.x+. You can't assume that just from a GitHub issue/comment. I regularly play on 1.7.10, 1.8.9 and Beta 1.7.3. But how would you have known by a GitHub issue/comment?

**Comment by shinji257** [comment]
So to try and subvert the argument here (or the insistence of fact) I'll suggest that rather than replace or depreciate MCEdit support then maybe we can get Amulet integration added to supplement?

It could be noted that MCEdit is beneficial for use on 1.12.2 and lower and that Amulet is useful for newer versions of Minecraft. 

I should also add that depreciated doesn't mean useless.  It just means it is no longer being updated or updates might only happen when there are major bugs on the existing supported versions.

**Comment by business-goose** [comment]
Yeah. It would be beneficial to have both MCEdit and Amulet integration 
so even when we're on legacy versions of Minecraft we can remove 
problematic chunks and blocks.

**Comment by xcfrg** [comment]
Seems like amulet still does not have support for command line arguments, this cannot be implemented until then

**Comment by business-goose** [comment]
> Seems like amulet still does not have support for command line arguments, this cannot be implemented until then

Ah, that's unfortunate, I guess we'll just have to wait until command line arguments for Amulet are implemented or someone finds a workaround.

**Comment by Shywizz** [comment]
Alright so after toying around there's an easy way out but you can't have both at the same time
i could provide a tutorial if this thread is still active enough

**Comment by xcfrg** [comment]
> Alright so after toying around there's an easy way out but you can't have both at the same time i could provide a tutorial if this thread is still active enough

What do you mean by "you can't have both at the same time"?

**Comment by zurgeg** [comment]
> ### Role
> 
> Sometimes, I incorrectly type a command in a command block and it can get very difficult to delete them.
> ### Suggestion
> 
> MultiMC should substitute the MCEdit editor with Amulet for post 1.12.2 versions.
> ### Benefit
> 
> If you open a world with MCEdit (at least on Windows) and it is post 1.12.2 it is impossible to get anything done with it.
> ### This suggestion is unique
> 
> I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

fyi, you can just copy `amulet_app.exe` to `mcedit.exe`, it doesn't open the world, but at least it opens it. I've also made [a fork](https://github.com/zurgeg/Amulet-Map-Editor) of Amulet that should be able to open the world, except I haven't checked yet.

The main reason this hasn't been implemented is probably because Amulet doesn't support loading maps from the commandline, but WorldEdit does

**Comment by mk-pmb** [comment]
Instead of shipping code for specific world editors, can we just make a configurable list of world-specific utilities?
Users could configure
* program name
* icon
* command line to be run, with slots for world name and current account nickname as displayed in the upper right corner
* list of environment variables to set for the launched program, with same slots as would be available for the command line
* (did I forget an important option?)
